src/main/main.ts

```
@@ -176,23 +180,29 @@ const createMainWindow = async () => {
-  // Load the Url or index.html file;
-  void mainWindow.loadURL(resolveHtmlPath(''))
-
   // Open devtools if the app is not packaged;
-  // if (isDebug) {
-  // 	mainWindow.webContents.openDevTools()
-  // }
+  if (isDebug) {
+    mainWindow.webContents.openDevTools()
+  }
+
+  splash.on('closed', () => (splash = null))
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)